docs/guides/qrng/qrng-remix/index.md

```diff
@@ -70,17 +70,13 @@ Airnode. To follow along, you can open the following contract in Remix and try
 deploying it yourself. This contract will be requesting the random number
 directly from the QRNG Provider.
 
-Head on to
-[Remix online IDE.](https://remix.ethereum.org/#url=https://raw.githubusercontent.com/api3-ecosystem/remix-contracts/master/contracts/QrngRequesterUpdated.sol&lang=en&optimize=false&runs=200&evmVersion=null&version=soljson-v0.8.18+commit.87f61d96.js)
-It should load up the `RemixQrngExample` contract.
-
-[Open in Remix](https://remix.ethereum.org/#url=https://raw.githubusercontent.com/api3-ecosystem/remix-contracts/master/contracts/QrngRequesterUpdated.sol&lang=en&optimize=false&runs=200&evmVersion=null&version=soljson-v0.8.18+commit.87f61d96.js)
+[Open in Remix](https://remix.ethereum.org/#url=https://raw.githubusercontent.com/api3-ecosystem/remix-contracts/master/contracts/QrngRequesterUpdated.sol)
```

Comment:
When I open these new URLs, I get an error because of a solidity compiler version mismatch. The previous URLs included parameters for the solidity version that avoided this. For UX reasons I think we should include these URL parameters.

Reply:
Right. I thought it changed the compiler version automatically, but I agree with this.